lib/physicians/schema.graphql

```diff
@@ -34,10 +34,10 @@ input UpsertPhysicianInput {
   addressLine1: String!
   addressLine2: String
   city: String!
-  province: Province!
+  province: Province
   postalCode: String!
   phone: String!
-  status: PhysicianStatus!
+  status: PhysicianStatus
```

Comment:
Could you please clarify why this and `province` above are being changed to nullable?

Reply:
@j-bunk

Reply:
That's a mistake on my end these should be changed back to nun-nullable 
When I was creating new physicians using the edit modal I was getting errors because those values were not specified in the modal and were set as null. This is now resolved with the db default values i think